Heakhafaw_1105.txt

```diff
@@ -0,0 +1,525 @@
+# Heakhafaw
+# -6,-4

```

Comment:
Data files shouldn't have comments that duplicate metadata. These are produced on-the-fly by the API for users of the site, but aren't stored as part of the data file in the site. See other files in res/Sectors/M1105 for examples